lava-v2-jobs-from-api.py

```diff
@@ -317,6 +318,8 @@ def main(args):
                         help="section in the KernelCI config file")
     parser.add_argument("--plans", nargs='+', required=True,
                         help="test plan to create jobs for")
+    parser.add_argument("--job_timeout", default="10", dest='job_timeout',
```

Comment:
Why add this as a command line argument in this script, which applies to all the jobs generated, rather than a YAML parameter in each test plan like I suggested on IRC?

Reply:
I didn't get what you exactly meant I should've asked, but is this a wrong one to do so or not preferable to do it this way ?